code/game/objects/structures.dm

```diff
@@ -240,13 +240,17 @@
 
 //Damage
 /obj/structure/proc/take_damage(dam)
-	if(!dam || CHECK_BITFIELD(resistance_flags, INDESTRUCTIBLE|UNACIDABLE))
```

Comment:
This change may produce unforeseen consequences we'll have to keep an eye for.

Reply:
Can you explain why the change then? If it could be breaking what was the need for it? (Its not obvious to me)

Reply:
We want the structure to be unacidable by xenos, but destroyable by marines.
That can't happen as it is.
This could break things because some things use the old format of assuming that unacidable is undestroyable. Meaning we'll need to fix more things in the long run, but it's the desirable way.